src/panel/common/commonActions.js

```diff
@@ -54,10 +76,27 @@ function isInvalidTags(tagArr) {
 	return false;
```

Comment:
* Placing `indicateInvalidTitleOrTags()` & `indicateInvalidId()` in `addNewList` makes for better SLAP (imo). Also avoids duplication of the line in 3 functions.
* Should `isInvalidTags` trim each tag before checking equality with `""`?

Nits:
* Incorrect indentation for the 3 functions above
* `if (i != index ...)` should be `if (i !== index ...)`